json_journal/charts.py

```diff
@@ -1,7 +1,12 @@
+'''Create pie chart'''
+
 def create_pie_chart(balances, axes):
+    '''balances -> dictionary of full account names and balances for each account,
+    axes -> specifed axes for plotting the pie chart.
+    Returns pie chart ready to be shown (n.b. plt.show needs to be called on this pie chart)'''
```

Comment:
I thing we can improve the formatting here. See some of the [numpy functions](https://github.com/numpy/numpy/blob/main/numpy/lib/arraypad.py#L33) for nice formatting examples.

I also think it would be a good idea to have similar headings to them eg `Parameters`, `Returns` where we can describe the structure of the (otherwise mysterious) dictionaries we're passing around.

Reply:
done - hopefully it's clearer now but open to feedback on how to make it even more clear